test/unittests/instructions_test.cpp

```diff
@@ -55,6 +56,8 @@ constexpr void validate_traits_of() noexcept
     // immediate_size
```

Comment:
Should `RJUMPV` included here somehow?

Reply:
Will add it but in RJUMPV commit

Reply:
Yup, I just wasn't even sure what is this function doing, but see it better now.